napari/_qt/dialogs/qt_notification.py

```diff
@@ -405,9 +405,11 @@ def _debug_tb(tb):
     QApplication.processEvents()
     QApplication.processEvents()
     with event_hook_removed():
-        print("Entering debugger. Type 'q' to return to napari.\n")
+        print(  # noqa: T201
+            "Entering debugger. Type 'q' to return to napari.\n"
+        )
```

Comment:
I imagine these probably must remain `print` otherwise they will conflict with the debugger stuff? I'm not sure how to test this actually.